test/Factories.hs

```diff
@@ -37,8 +37,8 @@ createUser ident = do
     Entity planId _ <- createFreePlan
 
     insertEntity User
-        { userName = "John Smith (" ++ ident ++ ")"
-        , userEmail = "john-" ++ ident ++ "@gmail.com"
+        { userName = "Dummy Login"
+        , userEmail = "dummy@example.com"
```

Comment:
This makes me nervous. If these values don't match those in `Model.User.dummyProfile`, tests will fail. However, I don't want to export `Profile(..)` and `dummyProfile` from `Model.User` for re-use. OK trade-off?